src/components/data/reports/FeeFineReport.js

```diff
@@ -0,0 +1,148 @@
+import { exportCsv } from '@folio/stripes/util';
+
+import { feeFineReportColumns } from '../../../constants';
+import { isEmpty, map } from 'lodash';
+import {
+  getFullName,
+  formatActionDescription,
+  formatCurrencyAmount,
+  formatDateAndTime,
+  getServicePointOfCurrentAction,
+  calculateRemainingAmount,
+} from '../../util';
+
+const getValue = (value) => value || '';
+
+const extractComments = (action) => {
+  const replacer = (comment, regexp) => comment.replace(regexp, '').trim();
+  const comments = {
+    actionInfoStaff: '',
+    actionInfoPatron: ''
+  };
+  const actionComments = action.comments ? action.comments.split('\n') : [];
+
+  actionComments.forEach((comment, index) => {
+    switch (index) {
+      case 0:
+        comments.actionInfoStaff = replacer(comment, /STAFF :/i);
+        break;
+      case 1:
+        comments.actionInfoPatron = replacer(comment, /PATRON :/i);
+    }
+  });
+
+  return comments;
+};
+
+class FeeFineReport {
+  constructor({ data, intl: { formatMessage, formatTime } }) {
+    // data model:
+    // data = {
+    //  feeFineActions: [] - GET: 'feefineactions?query=(userId==:{id}&limit=${MAX_RECORDS})',
+    //  accounts: [] - GET: 'accounts?query=(userId==:{id})&limit=${MAX_RECORDS}',
+    //  loans: [] - GET: 'circulation/loans?query=(userId==:{id})&limit=1000',
+    //  servicePoints: [] - props.okapi.currentUser.servicePoints
+    //  patronGroup: 'group' - string,
+    //  user: user - GET: 'users/:{id}',
+    // }
```

Comment:
Comment will be removed after finishing of [UXPROD-100](https://issues.folio.org/browse/UXPROD-100)

Reply:
Why remove the comment? It's really nice knowing what shape the data will have, and where it came from! Unless there is some specific reason it _shouldn't_ be present, I say leave it!

Reply:
Agree, will leave the comment